stader-cli/node/commands.go

```diff
@@ -368,6 +367,67 @@ func RegisterCommands(app *cli.App, name string, aliases []string) {
 					return nodeApproveSd(c)
 				},
 			},
+			{
+				Name:      "utilize-sd",
+				Aliases:   []string{"us"},
+				Usage:     "Utilize from the Utility pool.",
+				UsageText: "stader-cli node utilization-sd",
+				Flags: []cli.Flag{
+					cli.BoolFlag{
+						Name:  "yes, y",
+						Usage: "Automatically confirm SD utilize",
+					},
+				},
+				Action: utilizeSD,
+			},
+			{
+				Name:      "repay-sd",
+				Aliases:   []string{"rs"},
+				Usage:     "Repay the amount to reduce their utilization position",
+				UsageText: "stader-cli node repay-sd [options]",
+				Flags: []cli.Flag{
+					cli.StringFlag{
+						Name:  "amount, a",
+						Usage: "The amount of SD to repay",
+					},
+					cli.BoolFlag{
+						Name:  "yes, y",
+						Usage: "Automatically confirm SD repay",
+					},
+				},
+				Action: func(c *cli.Context) error {
+					if _, err := cliutils.ValidatePositiveEthAmount("sd repay amount", c.String("amount")); err != nil {
+						return err
+					}
+
+					// Run
+					return repaySD(c)
+				},
+			},
+			{
+				Name:      "repay-excess-sd",
+				Aliases:   []string{"res"},
+				Usage:     "Repay utilized SD using the excess SD collateral",
+				UsageText: "stader-cli node repay-excess-sd [options]",
+				Flags: []cli.Flag{
+					cli.StringFlag{
+						Name:  "amount, a",
+						Usage: "The amount of SD to repay excess",
+					},
```

Comment:
No need to pass in the amount in `repay-excess-sd` . it will repay the sd collateral which is above 200% back to the pool